pkg/addons/addon_ufw.go

```
@@ -0,0 +1,89 @@
+package addons
+
+import (
+	"fmt"
+
+	"github.com/xetys/hetzner-kube/pkg/clustermanager"
+)
+
+//UfwAddon installs ufw
+type UfwAddon struct {
+	masterNode   *clustermanager.Node
+	communicator clustermanager.NodeCommunicator
+	nodeCidr     string
+	nodes        []clustermanager.Node
+}
+
+//NewUfwAddon installs ufw to the cluster
+func NewUfwAddon(provider clustermanager.ClusterProvider, communicator clustermanager.NodeCommunicator) ClusterAddon {
+	masterNode, _ := provider.GetMasterNode()
```

Unhandled error, we prefer to never ignore errors ;)